packages/immutable-rootfs/30cos-immutable-rootfs/cos-mount-layout.sh

```diff
@@ -150,6 +150,7 @@ function mountOverlay {
     local workdir
     local fstab_line
 
+    [ ! -d "${mount}" ] && echo "Source mount path ${mount} does not exist!" && return
```

Comment:
We need to test against `${merged}` variable. Note this is executed inside initrd before switching the real root. At this stage the real root is at mounted at `/sysroot`, so in order to check if the directory is there we should test against `/sysroot/<dir>`, which is the `${merged}` value in this script.

In addition I think we will have to omit the `echo ` message. Note this method is expected to `echo` an fstab line of the mounted path, hence whatever we `echo` here will end up being part of the `/etc/fstab`.

Reply:
Thank you for your comments, I included all your suggestions